src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -975,84 +1235,68 @@ subroutine decomp_rate_constants_bgc(bounds, num_soilc, filter_soilc, &
 
       end if
 
-      if ( normalize_q10_to_century_tfunc ) then
-         ! scale all decomposition rates by a constant to compensate for offset between original CENTURY temp func and Q10
-         normalization_factor = (catanf(normalization_tref)/catanf_30) / (q10**((normalization_tref-25._r8)/10._r8))
-         do j = 1, nlevdecomp
-            do fc = 1,num_soilc
-               c = filter_soilc(fc)
-               t_scalar(c,j) = t_scalar(c,j) * normalization_factor
-            end do
-         end do
-      endif
-
-      if (use_vertsoilc) then
-         ! add a term to reduce decomposition rate at depth
-         ! for now used a fixed e-folding depth
-         do j = 1, nlevdecomp
-            do fc = 1,num_soilc
-               c = filter_soilc(fc)
-               depth_scalar(c,j) = exp(-zsoi(j)/decomp_depth_efolding)
-            end do
+!     ! TODO slevis: Commented out. Should I have kept these lines?
+!     if ( normalize_q10_to_century_tfunc ) then
+!        ! scale all decomposition rates by a constant to compensate for offset between original CENTURY temp func and Q10
+!        normalization_factor = (catanf(normalization_tref)/catanf_30) / (q10**((normalization_tref-25._r8)/10._r8))
+!        do j = 1, nlevdecomp
+!           do fc = 1,num_soilc
+!              c = filter_soilc(fc)
+!              t_scalar(c,j) = t_scalar(c,j) * normalization_factor
+!           end do
+!        end do
+!     endif
+
+      ! Term that reduces decomposition rate at depth
+      ! TODO slevis: Reduced code repetition by defining
+      !              depth_scalar(c,j) = 1.0_r8 in the else instead of
+      !              repeating the decomp_k lines.
```

Comment:
In this case I successfully reduced some code repetition.

Reply:
For now, all the changes below this line are due to the consolidation of repetitive code.